platform-sdk/swirlds-virtualmap/src/main/java/module-info.java

```diff
@@ -4,7 +4,7 @@
 open module com.swirlds.virtualmap {
     exports com.swirlds.virtualmap;
     exports com.swirlds.virtualmap.datasource;
-    // Currently, exported only for tests.
+    // is it really "internal"?
```

Comment:
Probably should resolve this. Reads like a "TODO". But I will not hold up the PR for this.

Reply:
I filed a low-priority issue about it: https://github.com/hashgraph/hedera-services/issues/6372